spring-boot/proxy/src/test/java/LocalApiKeyAndServiceApiManagerTest.java

```diff
@@ -0,0 +1,116 @@
+/*-
+ * #%L
+ * coatrack-proxy
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+import eu.coatrack.api.ApiKey;
+
+import eu.coatrack.proxy.security.AdminCommunicator;
+import eu.coatrack.proxy.security.LocalApiKeyAndServiceApiManager;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+public class LocalApiKeyAndServiceApiManagerTest extends LocalApiKeyAndServiceApiManager{
+
+    private final String someValidApiKeyValue = "someValidApiKeyValue";
+
+    private ApiKey apiKey;
+
+    private final long
+            oneHourInMillis = 1000 * 60 * 60,
+            oneDayInMillis = oneHourInMillis * 24;
+
+    private final Timestamp
+            now = new Timestamp(System.currentTimeMillis()),
```

Comment:
I am wondering if the new java date/time api could be used to improve code readability?!

Reply:
I used the LocalDateTime within LocalApiKeyAndServiceApiManager which works fine. However, the ApiKey class uses Date, therefore I used Timestamp (sub-class of Date) for legacy reasons.

Reply:
We should discuss in an independent issue if we should change the time implementation of ApiKey.